docs/view_components/view_specific_documentation.rst

```diff
@@ -0,0 +1,67 @@
+View Specific Documentation
+---------------------------
+
+This view-component lets the user access the specific sections of the documentation directly from within the tool UI.
```

Comment:
If I did not know what you want to describe, I don't think I could get it from this description. I think  you can use the concrete example and the main application of this view-component in the description: a tooltip for input parameters